src/logic/equiv/transfer_instance.lean

```diff
@@ -87,7 +87,7 @@ def mul_equiv (e : α ≃ β) [has_mul β] :
 begin
```

Comment:
`mul_equiv` is not a class so I'm not sure whether to make reducible. Same for the other non-classes.

Reply:
These don't have to be reducible. The linter/library note only refer to classes.

Reply:
Oh yes, I didn't read the library note carefully. It's necessary to mark the def reducible if it's used to define instances of classes that are arguments of other classes. Thanks!